apps/api_web/lib/api_web/controllers/facility_controller.ex

```diff
@@ -142,8 +156,16 @@ defmodule ApiWeb.FacilityController do
             )
 
             name(
+              :string,
```

Comment:
We'd just rename the field in the documentation: we don't need to maintain it for older versions.

Reply:
Fixed.